src/odemis/util/__init__.py

```diff
@@ -350,6 +350,33 @@ def normalize_rect(rect):
     return type(rect)((l, t, r, b))
 
 
+def is_point_in_rect(p, rect):
+    """
+    Check if a point is inside in a rectangle.
+
+    p (tuple of 2 floats): x, y coordinates of point in m
+    rect (tuple of 4 floats): l, t, r, b positions of rectangle in m
+    return (bool): True if point is in rectangle, False otherwise
+    """
+    l, t, r, b = rect
+    if l <= p[0] <= r and b <= p[1] <= t:
+        return True
+    else:
+        return False
+
+
+def expand_rect(rect, margin):
+    """
+    Expand a rectangle by a fixed margin.
+
+    rect (tuple of 4 floats): l, t, r, b positions of rectangle in m
+    margin (float): margin to increase rectangle by in m
+    return (tuple of 4 floats): l, t, r, b positions of adjusted rectangle in m
+    """
+    l, t, r, b = rect
+    return l - margin, t + margin, r + margin, b - margin
```

Comment:
I realise that this is different from what we normally do: l & t are the small ones and r & b the big ones. IOW, t < b. The names top/bottom are correct in pixel coordinates, but not physical coordinates (as Y is inverted, per convention). See the normalize_rect() function.
So, please follow this convention. You can also drop the "in m" in your docstring, as the rectangle computation is unit independent.
Please also change the `is_point_in_rect()` function.